tensorflow-core.yaml

```diff
@@ -52,7 +54,6 @@ environment:
       - perl
       - py3-astunparse
       - py3-pip
-      - py3-pybind11-dev
```

Comment:
This was pointing to an old/unmaintained version prior to the `-dev` merge of the package.